diff --git a/node_modules/@react-native-firebase/analytics/RNFBAnalytics.podspec b/node_modules/@react-native-firebase/analytics/RNFBAnalytics.podspec
index 0388463..0667673 100644
--- a/node_modules/@react-native-firebase/analytics/RNFBAnalytics.podspec
+++ b/node_modules/@react-native-firebase/analytics/RNFBAnalytics.podspec
@@ -24,6 +24,7 @@ Pod::Spec.new do |s|
   s.source              = { :git => "https://github.com/invertase/react-native-firebase.git", :tag => "v#{s.version}" }
   s.social_media_url    = 'http://twitter.com/invertaseio'
   s.ios.deployment_target = firebase_ios_target
+  s.tvos.deployment_target = firebase_ios_target
   s.macos.deployment_target = firebase_macos_target
   s.source_files        = 'ios/**/*.{h,m}'
 
